README.md

```diff
@@ -1 +1,31 @@
-# developer-portfolio
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Web Developer Portfolio
+
+> This is my portfolio as a web developer. It's the main tool in my Job Search, therefore the project I will spend the most hours working on. I hope you like it.
+
```

Comment:
[Optional]
- It looks good. Also, I will suggest adding screenshots to show people your work in one scroll. 👍